src/test/e2e/common.go

```diff
@@ -44,107 +42,41 @@ func getCLIName() string {
 	return binaryName
 }
 
-// cleanupAfterTest cleans up after a test run so that the next test can run in a clean environment. It needs to be
-// deferred at the beginning of each test. Example:
-//
-// func TestE2eFooBarBaz(t *testing.T) {
-//     defer e2e.cleanupAfterTest(t)
-//     doAllTheOtherStuff...
-// }
-func (e2e *ZarfE2ETest) cleanupAfterTest(t *testing.T) {
-	// Check if cluster is running and use Zarf to perform chart uninstallation
-	err := clusters.TryValidateClusterIsRunning()
-	if err == nil {
-		output, err := e2e.execZarfCommand("destroy", "--confirm", "--remove-components", "-l=trace")
-		require.NoError(t, err, output)
-	}
-
-	// Remove files created for the test
-	for _, filePath := range e2e.filesToRemove {
-		err = os.RemoveAll(filePath)
-		require.NoError(t, err, "unable to remove file when cleaning up after a test")
-	}
-	e2e.filesToRemove = []string{}
-
-	// Kill background processes spawned during the test
-	e2e.killBackgroundProcesses(t)
+// setup actions for each test
+func (e2e *ZarfE2ETest) setup(t *testing.T) {
+	t.Log("Test setup")
+	utils.ExecCommandWithContext(context.TODO(), true, "sh", "-c", "kubectl describe nodes |grep -A 99 Non\\-terminated")
+	utils.ExecCommandWithContext(context.TODO(), true, "lsof", "-iTCP", "-sTCP:LISTEN", "-n")
 }
 
-// Kill background processes spawned during the test
-func (e2e *ZarfE2ETest) killBackgroundProcesses(t *testing.T) {
+// teardown actions for each test
+func (e2e *ZarfE2ETest) teardown(t *testing.T) {
+	t.Log("Test teardown")
+	// Kill background processes spawned during the test
 	for _, cmd := range e2e.cmdsToKill {
 		if cmd.Process != nil {
-			err := cmd.Process.Kill()
-			require.NoError(t, err, "unable to kill background cmd when cleaning up after a test")
+			if err := cmd.Process.Kill(); err != nil {
+				t.Logf("unable to kill process: %v", err)
```

Comment:
Just logging this instead of failing fast?

Reply:
I was torn on this one but it's more a ci env issue than zarf issue I think if this occurs.  I haven't seen it occur yet fwiw.

Reply:
Fair enough. If it ends up being a problem we'll see it almost immediately later anyways.